pyquil/quil.py

```diff
@@ -851,7 +880,11 @@ def address_qubits(program, qubit_mapping=None):
     return new_program
 
 
-def _get_label(placeholder, label_mapping, label_i):
+def _get_label(
+    placeholder: Union[Label, LabelPlaceholder],
+    label_mapping: Dict[Union[Label, LabelPlaceholder], Label],
+    label_i: int,
+) -> Tuple[Label, Dict[Union[Label, LabelPlaceholder], Label], int]:
```

Comment:
I assume this was done to satisfy mypy about some error, but it seems like `Union[Label, LabelPlaceholder]` should be replaced by just `LabelPlaceholder` everywhere in this function signature, then update the call sites if necessary to satisfy mypy.

Reply:
Replaced all `Union[Label, LabelPlaceholder]` with `LabelPlaceholder` in 5th commit.